other/edge-server/README.md

```diff
@@ -242,15 +265,15 @@ Install the required dependencies:
 npm install
 ```
 
-Add a `.env` file inside the `node-server` directory with the details 
+Add a `.env` file inside the `node-server` directory with the details
 required to run the Express Server.
 
 ```env
 # This URI string is for Atlas Edge Server
 EDGE_SERVER_URI="mongodb://localhost:27021"
 
 # This port is for the Node.js server
-PORT=5000
+PORT=5055
 ```
 
 Then, run the Express server:
```

Comment:
We can probably remove these lines about running the Express Server since that is done now at the higher level.

Reply:
Can't comment below, but we should update the path in ln 269/292 to show that `node-server` is in the `client/` directory, and in ln 273/296 to say instead "at the root of your `client/node-server` directory.